hazelcast/src/test/java/com/hazelcast/spi/utils/RestClientTest.java

```diff
@@ -50,6 +56,8 @@
 import java.nio.file.Paths;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+@RunWith(HazelcastParallelClassRunner.class)
```

Comment:
are you sure about that? sounds a bit like IT and ITs are IMHO rarely parallel

Reply:
This should be OK. Most of the tests in the class are cheap HTTP client calls. The two TLS 1.3 tests open server sockets, but still no big deal.